cluster/run_tests.sh

```diff
@@ -0,0 +1,3 @@
+pushd tests
```

Comment:
KUBEVIRT_PATH is currently only used to make  cluster/kubectl.sh work from everywhere, so good enought for me like it is now.